cpp/CMakeLists.txt

```diff
@@ -293,6 +293,12 @@ endif(Boost_FOUND)
 ###################################################################################################
 # - RMM -------------------------------------------------------------------------------------------
 
+if(NOT SPDLOG_ACTIVE_LEVEL)
```

Comment:
Doesn't just setting -DSPDLOG_ACTIVE_LEVEL on the CMake command line automatically cause that to be set on the compiler command line?

If not, then at least use a more generic name matching what is in the RMM CMake: LOGGING_LEVEL

Reply:
> Doesn't just setting -DSPDLOG_ACTIVE_LEVEL on the CMake command line automatically cause that to be set on the compiler command line?

No, I tried that initially per the offline discussion and no dice.  I believe one has to explicitly handle this, either via variables that are expected to be defined only the command line and always appended (e.g.: `EXTRA_CXX_FLAGS`), or via specific settings/options that are handled in cmake logic.

I'll model this to be similar to the RMM cmake naming and logic for consistency.

Reply:
I've updated this to use the same `LOGGING_LEVEL` cmake variable used by the RMM build.  Thanks for the pointer, @harrism!